.github/workflows/main-build.yml

```diff
@@ -17,26 +17,31 @@ jobs:
             build_directory: java
             build_command: ./build.sh
             terraform_directory: integration-tests/java/aws-sdk/agent
+            expected_template: adot/utils/expected-templates/java-awssdk-agent.json
```

Comment:
If you want you can just use `${{ matrix.name }}` instead of another variable that seems to map 1:1 with it, but not a big deal